frontend/src/components/CommunityConfiguration/communityConfigLayout.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import PluginCard from '../PluginCards/pluginCard'
+import { Container, Row, Col, Button} from 'reactstrap';
+import { Pagination, PaginationItem, PaginationLink, Spinner } from 'reactstrap';
+import ConfigurationCard from './configurationCard';
+
+class communityConfigLayout extends React.Component {
+
+    state = {
+        search:"",
+        data: [
```

Comment:
Is there a way to dynamically load these?  That would make it easier to work with here.  At the same time, I wonder if there's a way to go through and add tags for these pieces for search.  That could be another ticket.

Reply:
Yeah working on getting these to load dynamically